app/src/main/res/values/strings.xml

```diff
@@ -250,17 +244,17 @@
     <!-- This is the the button description on the device security dialog box which routes to system security settings. -->
     <string name="set_up_now">Set Up Now</string>
 
-    <!-- This header appears above the Privacy page of the mozilla-lockbox website. -->
+    <!-- This header appears above the Privacy page of the application website. -->
     <string name="privacy">Privacy</string>
 
-    <!-- This is the prompt when autofill is triggered, and Firefox Lockbox is locked. -->
-    <string name="autofill_authenticate_cta">Unlock Firefox Lockbox</string>
+    <!-- This is the prompt when autofill is triggered and the application is locked. %1$s will be replaced with application name -->
+    <string name="autofill_authenticate_cta">Unlock %1$s</string>
```

Comment:
@sashei this appears as "Unlock %1$s" on the system-provided unlock dialog above "Enter your device Pattern, PIN, or Password"

Reply:
😬we're gonna have conflicts with the lockedpresenter work but I'll go ahead + do this for now.